neural_lam/vis.py

```diff
@@ -87,7 +87,7 @@ def plot_prediction(
         1,
         2,
         figsize=(13, 7),
-        subplot_kw={"projection": data_config.coords_projection()},
```

Comment:
I think this will be fixed by @sadamov in #40, maybe we should wait for that to be merged? What do you think @sadamov?

Reply:
yes lets fix the bugs I introduced, I also had to manually fix this in multiple branches which is annoying. @leifdenby could you give a final review on #40 and then @SimonKamuk merge this PR-branch with main later.

Reply:
Should be fixed now with the merge of #17

Reply:
> Should be fixed now with the merge of #17

Ok, sounds good. Maybe merge upstream into your branch @SimonKamuk and push this branch to github again?